### ISSUE TITLE
Increase touch target of SearchDialogFragment views

### ISSUE THREAD

**Issue by Mugurell** [mozilla-mobile/fenix#24461]
![image](https://user-images.githubusercontent.com/11428869/160153845-7e2a6c39-9827-4fae-ae23-0990bb095396.png)

According to the [Android accessibility recommendations](https://support.google.com/accessibility/android/answer/7101858) touch targets should be at least 48x48 dp.
For this [our increaseTapArea](https://github.com/mozilla-mobile/fenix/blob/47f55924e1ff23ee56a5db0af1b15f86a33a5f8b/app/src/main/java/org/mozilla/fenix/ext/View.kt#L22) extension function can be used.

**Comment by jknair0** [comment]
Hi! Can pick this up?

**Comment by Mugurell** [comment]
> Hi! Can pick this up?

Would be great to! Thank you!

**Comment by Mugurell** [comment]
@ QA Please do another scan of the buttons shown in the search screen.
There should be an improvement. For the remaining issues we'll open new tickets.

**Comment by SoftVision-LorandJanos** [comment]
Verified as fixed on the latest Nightly 101.0a1 (2022-04-13) build.
Device used: Google Pixel 4 (Android 12).
Closing the ticket as fixed.
